lib/litmus_paper/dependency/script.rb

```diff
@@ -1,3 +1,5 @@
+require 'open3'
```

Comment:
if the intent is to remove deps you can call `Kernel.spawn` directly and create 2 pipes one for `stderr` and one for `stdout` which is what `Open3.popen*` functions do